views/surveys/pullAndParseSurveys.py

```diff
@@ -4,12 +4,13 @@
 import requests
 from bs4 import BeautifulSoup
 
-database = "Actual"
-username = "postgres"
-password = "password"
-address = "10.11.12.24"
-googleEmail = "pokcpcapep@gmail.com"
-googlePassword = "Marist1234"
+# config.ini information is passed in as arguments from php shell_exec
```

Comment:
This lets us hide things like the raw text gmail login and password, storing them in `config.ini` which is on the `.gitignore` list.